recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
```

Comment:
Yeah, I want to move away from the string because string comparison is being eliminated in `conda` IIRC. Whereas using this PEP440 compliant versioneer style versioning will work great with `conda` (even with old versions).